modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/java/com/liferay/multi/factor/authentication/fido2/web/internal/constants/MFAFIDO2WebKeys.java

```diff
@@ -19,6 +19,10 @@
  */
 public class MFAFIDO2WebKeys {
 
+	public static final String MFA_FIDO2_ASSERTION_REQUEST = "assertionRequest";
+
+	public static final String MFA_FIDO2_PKCC_OPTIONS = "pkccOptions";
+
```

Comment:
hey @arthurchan35, 

it looks like Brian wants the value of the constant to match the constant, at least initially. 

so it would need to look `"MFA_FIDO2_PKCC_OPTIONS"`